internal/stack/serverless.go

```diff
@@ -145,6 +150,11 @@ func (sp *serverlessProvider) currentProjectWithClientsAndFleetEndpoint(config C
 	}
 	project.Endpoints.Fleet = fleetURL
 
+	err = project.AddLogstashFleetOutput(sp.kibanaClient)
+	if err != nil {
+		return nil, fmt.Errorf("failed to add logstash fleet output: %w", err)
+	}
+
```

Comment:
>This call would re-add the Output to the Serverless project (doing a Post query), is that right?

Yes, it adds a `logstash` fleet output using a post request.

>This currentProjectWithClientsAndFleetEndpoint method is used to fill some missing information (Fleet endpoints) after creating the Serverless Project (for instance to be used in the elastic-package stack status command) . Would it be ok to just add the logstash output (POST query) in the createProject() method/request ?

I was a bit unsure of this too.. I add here because the FleetURL is being added here and clients are created. So I was thinking if the outputs already added be present or not?

Reply:
> > This currentProjectWithClientsAndFleetEndpoint method is used to fill some missing information (Fleet endpoints) after creating the Serverless Project (for instance to be used in the elastic-package stack status command) . Would it be ok to just add the logstash output (POST query) in the createProject() method/request ?
> 
> I was a bit unsure of this too.. I add here because the FleetURL is being added here and clients are created. So I was thinking if the outputs already added be present or not?

As the Fleet Output has been created in the `createProject()` method, IIRC there should not be needed to repeat that call again in `currentProjectWithClientsAndFleetEndpoint()` method.
Currently, there are two usages of this method:
- After creating the project to fill the endpoints
  https://github.com/elastic/elastic-package/blob/5cd61115989b3e16d41bd223b3c129436aa1a221/internal/stack/serverless.go#L258 
- In status command, to get the right Fleet endpoints (serverless project should be already created):
  https://github.com/elastic/elastic-package/blob/5cd61115989b3e16d41bd223b3c129436aa1a221/internal/stack/serverless.go#L389

In both locations, the serverless project has been already created and that Fleet output should be already added/present.

Reply:
Then I just add the Outputs during `createProject()` call